2012/04/04 10:44:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "patrick" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('patrick')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=patrick
HTTP_REFERER=http://localhost/yii/ayiitest/
---
2012/04/04 10:44:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "patrick".' in /var/www/yii/framework/web/CController.php:484
Stack trace:
#0 /var/www/yii/framework/web/CController.php(271): CController->missingAction('patrick')
#1 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('patrick')
#2 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/patrick')
#3 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#4 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/patrick
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/index
---
2012/04/04 11:23:55 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "test" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('test')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=test
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/index
---
2012/04/04 11:23:56 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "test" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('test')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=test
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/page&view=test
---
2012/04/04 11:45:12 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Maps" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('Maps')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=Maps
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/index
---
2012/04/04 11:45:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Distance Matrix" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('Distance Matrix')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=Distance+Matrix
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/page&view=Maps
---
2012/04/04 11:45:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Maps" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('Maps')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=Maps
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/page&view=Distance+Matrix
---
2012/04/04 11:46:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The requested view "Maps" was not found.' in /var/www/yii/framework/web/actions/CViewAction.php:113
Stack trace:
#0 /var/www/yii/framework/web/actions/CViewAction.php(124): CViewAction->resolveView('Maps')
#1 /var/www/yii/framework/web/actions/CAction.php(75): CViewAction->run()
#2 /var/www/yii/framework/web/CController.php(309): CAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CViewAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CViewAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('page')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/page')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/page&view=Maps
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/page&view=Distance+Matrix
---
2012/04/04 15:07:42 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/crud/CrudCode.php(44): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): CrudCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/crud')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/crud
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii/default/index
---
2012/04/04 15:08:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii/default/index
---
2012/04/04 15:09:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[28000] [1045] Access denied for user 'root'@'localhost' (using password: NO)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii/default/index
---
2012/04/04 15:10:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii/default/index
---
2012/04/04 15:11:37 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii/default/index
---
2012/04/04 15:13:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2005] Unknown MySQL server host 'localhost/yii/ayiitest/protected/data' (1)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii/default/index
---
2012/04/04 15:13:28 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:18:17 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: could not find driver' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
---
2012/04/04 15:24:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:36:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database ' /../data/testdrive.db'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:37:23 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2005] Unknown MySQL server host 'localhost/yii/ayiitest/protected/data' (1)' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:39:31 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:19 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:20 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:40:22 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 15:59:30 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[42000] [1049] Unknown database 'testdrive'' in /var/www/yii/framework/db/CDbConnection.php:382
Stack trace:
#0 /var/www/yii/framework/db/CDbConnection.php(331): CDbConnection->open()
#1 /var/www/yii/framework/db/CDbConnection.php(309): CDbConnection->setActive(true)
#2 /var/www/yii/framework/base/CModule.php(388): CDbConnection->init()
#3 /var/www/yii/framework/base/CModule.php(104): CModule->getComponent('db')
#4 /var/www/yii/framework/gii/generators/model/ModelCode.php(54): CModule->__get('db')
#5 /var/www/yii/framework/web/CFormModel.php(40): ModelCode->init()
#6 /var/www/yii/framework/gii/CCodeGenerator.php(152): CFormModel->__construct()
#7 /var/www/yii/framework/gii/CCodeGenerator.php(64): CCodeGenerator->prepare()
#8 /var/www/yii/framework/web/actions/CInlineAction.php(50): CCodeGenerator->actionIndex()
#9 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#10 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#11 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('')
#13 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('gii/model')
#14 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#15 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=gii/model
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=gii
---
2012/04/04 16:11:16 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.user' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `user`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:47:19 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:47:54 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:48:29 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:48:42 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:49:03 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_comment' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_comment`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:50:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 16:50:46 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:00:47 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:01:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:01:26 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:01:33 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_post' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_post`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:01:40 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_comment' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_comment`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:02:11 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_comment' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_comment`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:02:18 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_comment' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_comment`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:02:20 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_comment' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_comment`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/04 17:02:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'testdrive.tbl_comment' doesn't exist. The SQL statement executed was: SHOW COLUMNS FROM `tbl_comment`.
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:21:04 [error] [exception.CException] exception 'CException' with message 'Property "TestForm.body" is not defined.' in /var/www/yii/framework/base/CComponent.php:131
Stack trace:
#0 /var/www/yii/framework/web/helpers/CHtml.php(2044): CComponent->__get('body')
#1 /var/www/yii/framework/web/helpers/CHtml.php(1258): CHtml::resolveValue(Object(TestForm), 'body')
#2 /var/www/yii/framework/web/widgets/CActiveForm.php(620): CHtml::activeTextArea(Object(TestForm), 'body', Array)
#3 /var/www/yii/ayiitest/protected/views/site/test.php(20): CActiveForm->textArea(Object(TestForm), 'body', Array)
#4 /var/www/yii/framework/web/CBaseController.php(127): require('/var/www/yii/ay...')
#5 /var/www/yii/framework/web/CBaseController.php(96): CBaseController->renderInternal('/var/www/yii/ay...', Array, true)
#6 /var/www/yii/framework/web/CController.php(870): CBaseController->renderFile('/var/www/yii/ay...', Array, true)
#7 /var/www/yii/framework/web/CController.php(783): CController->renderPartial('test', Array, true)
#8 /var/www/yii/ayiitest/protected/controllers/SiteController.php(115): CController->render('test', Array)
#9 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#10 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#11 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#12 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('test')
#14 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/test')
#15 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#16 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/contact
---
2012/04/05 18:28:09 [warning] [application] Failed to set unsafe attribute "data" of "TestForm".
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (111)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:32:32 [warning] [application] Failed to set unsafe attribute "data" of "TestForm".
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (111)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:32:45 [warning] [application] Failed to set unsafe attribute "data" of "TestForm".
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (111)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:32:46 [warning] [application] Failed to set unsafe attribute "data" of "TestForm".
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (111)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:32:47 [warning] [application] Failed to set unsafe attribute "data" of "TestForm".
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (111)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:34:57 [warning] [application] Failed to set unsafe attribute "data" of "TestForm".
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (111)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 18:47:46 [error] [php] Use of undefined constant php - assumed 'php' (/var/www/yii/ayiitest/protected/views/site/test.php:21)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(870): SiteController->renderFile()
#1 /var/www/yii/framework/web/CController.php(783): SiteController->renderPartial()
#2 /var/www/yii/ayiitest/protected/controllers/SiteController.php(121): SiteController->render()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#4 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#6 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/views/site/test.php (21)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (121)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 19:25:53 [error] [php] Object of class HTTP_Request2_Response could not be converted to string (/var/www/yii/ayiitest/protected/views/site/test.php:24)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(870): SiteController->renderFile()
#1 /var/www/yii/framework/web/CController.php(783): SiteController->renderPartial()
#2 /var/www/yii/ayiitest/protected/controllers/SiteController.php(140): SiteController->render()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#4 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#6 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/views/site/test.php (24)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (140)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 19:26:15 [error] [php] Object of class HTTP_Request2_Response could not be converted to string (/var/www/yii/ayiitest/protected/views/site/test.php:24)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(870): SiteController->renderFile()
#1 /var/www/yii/framework/web/CController.php(783): SiteController->renderPartial()
#2 /var/www/yii/ayiitest/protected/controllers/SiteController.php(140): SiteController->render()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#4 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#6 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/views/site/test.php (24)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (140)
in /var/www/yii/ayiitest/index.php (13)
2012/04/05 19:26:47 [error] [php] Object of class HTTP_Request2_Response could not be converted to string (/var/www/yii/ayiitest/protected/views/site/test.php:24)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(870): SiteController->renderFile()
#1 /var/www/yii/framework/web/CController.php(783): SiteController->renderPartial()
#2 /var/www/yii/ayiitest/protected/controllers/SiteController.php(140): SiteController->render()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#4 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#6 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/views/site/test.php (24)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (140)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 14:48:48 [error] [php] include(integer.php): failed to open stream: No such file or directory (/var/www/yii/framework/YiiBase.php:418)
Stack trace:
#0 /var/www/yii/framework/validators/CValidator.php(171): import()
#1 /var/www/yii/framework/base/CModel.php(285): createValidator()
#2 /var/www/yii/framework/base/CModel.php(259): TestForm->createValidators()
#3 /var/www/yii/framework/base/CModel.php(302): TestForm->getValidators()
#4 /var/www/yii/framework/web/helpers/CHtml.php(1185): TestForm->isAttributeRequired()
#5 /var/www/yii/framework/web/widgets/CActiveForm.php(560): activeLabelEx()
#6 /var/www/yii/ayiitest/protected/views/site/test.php(23): CActiveForm->labelEx()
#7 /var/www/yii/framework/web/CBaseController.php(127): require()
#8 /var/www/yii/framework/web/CBaseController.php(96): SiteController->renderInternal()
#9 /var/www/yii/framework/web/CController.php(870): SiteController->renderFile()
#10 /var/www/yii/framework/web/CController.php(783): SiteController->renderPartial()
#11 /var/www/yii/ayiitest/protected/controllers/SiteController.php(140): SiteController->render()
#12 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#13 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#14 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#15 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#16 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#17 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#18 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#19 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/views/site/test.php (23)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (140)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 14:58:51 [error] [php] Undefined variable: request (/var/www/yii/ayiitest/protected/controllers/SiteController.php:122)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#1 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#2 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#3 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#4 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#5 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (122)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 15:10:01 [error] [php] include(data.php): failed to open stream: No such file or directory (/var/www/yii/framework/YiiBase.php:418)
Stack trace:
#0 /var/www/yii/ayiitest/protected/models/TestForm.php(36): spl_autoload_call()
#1 /var/www/yii/framework/validators/CInlineValidator.php(43): TestForm->authenticate()
#2 /var/www/yii/framework/validators/CValidator.php(197): CInlineValidator->validateAttribute()
#3 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#4 /var/www/yii/ayiitest/protected/controllers/SiteController.php(119): TestForm->validate()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#6 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#7 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#8 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#9 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#10 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#11 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#12 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (36)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (119)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 15:10:20 [error] [php] include(data.php): failed to open stream: No such file or directory (/var/www/yii/framework/YiiBase.php:418)
Stack trace:
#0 /var/www/yii/ayiitest/protected/models/TestForm.php(36): spl_autoload_call()
#1 /var/www/yii/framework/validators/CInlineValidator.php(43): TestForm->authenticate()
#2 /var/www/yii/framework/validators/CValidator.php(197): CInlineValidator->validateAttribute()
#3 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#4 /var/www/yii/ayiitest/protected/controllers/SiteController.php(119): TestForm->validate()
#5 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#6 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#7 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#8 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#9 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#10 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#11 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#12 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (36)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (119)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 16:28:12 [error] [php] Undefined variable: pattern (/var/www/yii/ayiitest/protected/models/TestForm.php:34)
Stack trace:
#0 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#1 /var/www/yii/ayiitest/protected/controllers/SiteController.php(119): TestForm->validate()
#2 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#3 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#4 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#5 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#6 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#7 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#8 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#9 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (34)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (119)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 16:29:12 [error] [php] preg_match() expects parameter 1 to be string, array given (/var/www/yii/ayiitest/protected/models/TestForm.php:34)
Stack trace:
#0 /var/www/yii/framework/validators/CValidator.php(197): CInlineValidator->validateAttribute()
#1 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#2 /var/www/yii/ayiitest/protected/controllers/SiteController.php(119): TestForm->validate()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#4 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#6 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (34)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (119)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 16:30:21 [error] [php] Undefined variable: param (/var/www/yii/ayiitest/protected/models/TestForm.php:34)
Stack trace:
#0 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#1 /var/www/yii/ayiitest/protected/controllers/SiteController.php(119): TestForm->validate()
#2 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#3 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#4 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#5 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#6 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#7 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#8 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#9 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (34)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (119)
in /var/www/yii/ayiitest/index.php (13)
2012/04/17 17:24:02 [error] [php] preg_match(): Unknown modifier ')' (/var/www/yii/ayiitest/protected/models/TestForm.php:35)
Stack trace:
#0 /var/www/yii/framework/validators/CValidator.php(197): CInlineValidator->validateAttribute()
#1 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#2 /var/www/yii/ayiitest/protected/controllers/SiteController.php(119): TestForm->validate()
#3 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#4 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#5 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#6 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#7 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#8 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#9 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#10 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (35)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (119)
in /var/www/yii/ayiitest/index.php (13)
2012/04/20 15:21:06 [error] [php] Undefined variable: data (/var/www/yii/ayiitest/protected/models/TestForm.php:38)
Stack trace:
#0 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#1 /var/www/yii/ayiitest/protected/controllers/SiteController.php(120): TestForm->validate()
#2 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#3 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#4 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#5 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#6 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#7 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#8 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#9 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (38)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (120)
in /var/www/yii/ayiitest/index.php (13)
2012/04/20 15:32:05 [error] [php] Undefined variable: request (/var/www/yii/ayiitest/protected/controllers/SiteController.php:125)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#1 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#2 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#3 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#4 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#5 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (125)
in /var/www/yii/ayiitest/index.php (13)
2012/04/20 15:34:20 [error] [php] Undefined variable: data (/var/www/yii/ayiitest/protected/models/TestForm.php:39)
Stack trace:
#0 /var/www/yii/framework/base/CModel.php(158): CInlineValidator->validate()
#1 /var/www/yii/ayiitest/protected/controllers/SiteController.php(120): TestForm->validate()
#2 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#3 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#4 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#5 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#6 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#7 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#8 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#9 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (39)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (120)
in /var/www/yii/ayiitest/index.php (13)
2012/04/20 15:46:48 [error] [php] Undefined variable: data (/var/www/yii/ayiitest/protected/models/TestForm.php:41)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#1 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#2 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#3 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#4 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#5 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#6 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (41)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (122)
in /var/www/yii/ayiitest/index.php (13)
2012/04/20 15:47:30 [error] [exception.CException] exception 'CException' with message 'Property "SiteController.data" is not defined.' in /var/www/yii/framework/base/CComponent.php:131
Stack trace:
#0 /var/www/yii/ayiitest/protected/controllers/SiteController.php(123): CComponent->__get('data')
#1 /var/www/yii/framework/web/actions/CInlineAction.php(50): SiteController->actionTest()
#2 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams(Array)
#3 /var/www/yii/framework/web/CController.php(287): CController->runAction(Object(CInlineAction))
#4 /var/www/yii/framework/web/CController.php(266): CController->runActionWithFilters(Object(CInlineAction), Array)
#5 /var/www/yii/framework/web/CWebApplication.php(276): CController->run('test')
#6 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController('site/test')
#7 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#8 /var/www/yii/ayiitest/index.php(13): CApplication->run()
#9 {main}
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
HTTP_REFERER=http://localhost/yii/ayiitest/index.php?r=site/test
---
2012/04/20 15:50:35 [error] [php] Undefined variable: data (/var/www/yii/ayiitest/protected/models/TestForm.php:43)
Stack trace:
#0 /var/www/yii/framework/web/CController.php(309): CInlineAction->runWithParams()
#1 /var/www/yii/framework/web/CController.php(287): SiteController->runAction()
#2 /var/www/yii/framework/web/CController.php(266): SiteController->runActionWithFilters()
#3 /var/www/yii/framework/web/CWebApplication.php(276): SiteController->run()
#4 /var/www/yii/framework/web/CWebApplication.php(135): CWebApplication->runController()
#5 /var/www/yii/framework/base/CApplication.php(162): CWebApplication->processRequest()
#6 /var/www/yii/ayiitest/index.php(13): CWebApplication->run()
REQUEST_URI=/yii/ayiitest/index.php?r=site/test
in /var/www/yii/ayiitest/protected/models/TestForm.php (43)
in /var/www/yii/ayiitest/protected/controllers/SiteController.php (122)
in /var/www/yii/ayiitest/index.php (13)
